regression-test/data/nereids_ssb_shape_sf100_p0/shape/q3.4.out

```diff
@@ -8,23 +8,22 @@ PhysicalResultSink
 ----------PhysicalDistribute
 ------------hashAgg[LOCAL]
 --------------PhysicalProject
-----------------hashJoin[INNER_JOIN] hashCondition=((lineorder.lo_custkey = customer.c_custkey)) otherCondition=() build RFs:RF2 c_custkey->[lo_custkey]
-------------------PhysicalDistribute
---------------------PhysicalProject
-----------------------hashJoin[INNER_JOIN] hashCondition=((lineorder.lo_orderdate = dates.d_datekey)) otherCondition=() build RFs:RF1 d_datekey->[lo_orderdate]
-------------------------hashJoin[INNER_JOIN] hashCondition=((lineorder.lo_suppkey = supplier.s_suppkey)) otherCondition=() build RFs:RF0 s_suppkey->[lo_suppkey]
---------------------------PhysicalProject
-----------------------------PhysicalOlapScan[lineorder] apply RFs: RF0 RF1 RF2
---------------------------PhysicalDistribute
-----------------------------PhysicalProject
-------------------------------filter(s_city IN ('UNITED KI1', 'UNITED KI5'))
---------------------------------PhysicalOlapScan[supplier]
+----------------hashJoin[INNER_JOIN] hashCondition=((lineorder.lo_orderdate = dates.d_datekey)) otherCondition=() build RFs:RF2 d_datekey->[lo_orderdate]
+------------------PhysicalProject
+--------------------hashJoin[INNER_JOIN] hashCondition=((lineorder.lo_custkey = customer.c_custkey)) otherCondition=() build RFs:RF1 c_custkey->[lo_custkey]
+----------------------hashJoin[INNER_JOIN] hashCondition=((lineorder.lo_suppkey = supplier.s_suppkey)) otherCondition=() build RFs:RF0 s_suppkey->[lo_suppkey]
+------------------------PhysicalProject
+--------------------------PhysicalOlapScan[lineorder] apply RFs: RF0 RF1 RF2
 ------------------------PhysicalDistribute
 --------------------------PhysicalProject
-----------------------------filter((dates.d_yearmonth = 'Dec1997'))
-------------------------------PhysicalOlapScan[dates]
+----------------------------filter(s_city IN ('UNITED KI1', 'UNITED KI5'))
+------------------------------PhysicalOlapScan[supplier]
+----------------------PhysicalDistribute
+------------------------PhysicalProject
+--------------------------filter(c_city IN ('UNITED KI1', 'UNITED KI5'))
+----------------------------PhysicalOlapScan[customer]
 ------------------PhysicalDistribute
 --------------------PhysicalProject
-----------------------filter(c_city IN ('UNITED KI1', 'UNITED KI5'))
-------------------------PhysicalOlapScan[customer]
+----------------------filter((dates.d_yearmonth = 'Dec1997'))
```

Comment:
join order is changed, pls make sure the perf doesn't have regression

Reply:
improved from 0.13 sec to 0.11 sec.
this join order change has no impact on performance. the improvement comes from the broadcast join.